### ISSUE TITLE
Question: Example usage of get/setWifiConfig

### ISSUE THREAD

**Issue by scriptsrc** [sbidy/pywizlight#72]
Hey team,

Thanks a ton for this library and for your efforts to get it merged into HomeAssistant.

I have ~70 wiz bulbs and I'm starting to move them to a new vlan/ssid on my network. This is quite painful to do manually. I'm hoping to use the get/setWifiConfig UDP methods from your readme.

**Is there an example piece of python on how to do this?** (I've read the readme but I'm not quite sure how to use it just yet)

BTW - I love how you wrote this library to be async.

Thanks for any help.  I'm now a sponsor too!

**Comment by sbidy** [comment]
Unfortunately you can't set the bulb config via UDP packages. The function is available but not used for setting the config.

**Comment by scriptsrc** [comment]
Thanks for the quick response

Just to confirm - the setWifiConfig method mentioned in the readme does not
work. Is that correct?

**Comment by sbidy** [comment]
Yes - I documented all function which are exists on the bulb FW (https://github.com/sbidy/pywizlight#set-functions).
But not all of them works as "expected".

**Comment by BonnieSoftware** [comment]
setWifiConfig works when the mobile phone is connected to the light bulbs using the light bulbs own wifi hotspot, once the lightbulb is not in pairing mode anymore the hotspot is turned off and the light bulb connects to the wifi network using the details provided in the setWifiConfig  datagram, this is the same way Lifx bulbs work too.

**Comment by apizzi69** [comment]
Hi, Just curious if someone can explain how WiZ bulbs are discovered via hotspot and brought into the Wifi network.. Can anyone explain that process and what would need to be done by me?
